src/transaction/locator_sr.c

```diff
@@ -4647,6 +4654,13 @@ locator_check_primary_key_update (THREAD_ENTRY * thread_p, OR_INDEX * index, DB_
 
 	  do
 	    {
+#if !defined (SERVER_MODE)
+	      /* In not SERVER mode, btree_physical_delete() is called. Access to the deallocated page must be prevented. 
+	       * In locator_add_or_remove_index_internal(), you can see that it operates differently according to the "use_mvcc" variable.
+	       * If not SERVER_MODE, "use_mvcc" is always false in that function. And in those conditions, problems arise.
+	       */
+	      bt_scan.is_key_partially_processed = false;
+#endif
```

Comment:
this code would be better to move the front of btree_range_scan.

Reply:
For the same reason as above, I will keep the current code.

Reply:
I misunderstood.
Fixed to apply your comments.